openmc/data/multipole.py

```diff
@@ -661,3 +651,48 @@ def __call__(self, E, T):
 
         fun = np.vectorize(lambda x: self._evaluate(x, T))
         return fun(E)
+
+    def to_hdf5(self, path, libver='earliest'):
+        """Export windowed multipole data to an HDF5 file.
+
+        Parameters
+        ----------
+        path : str
+            Path to write HDF5 file to
+        libver : {'earliest', 'latest'}
+            Compatibility mode for the HDF5 file. 'latest' will produce files
+            that are less backwards compatible but have performance benefits.
+
+        """
+
+        # Open file and write version.
+        f = h5py.File(path, 'w', libver=libver)
+        f.create_dataset('version', (1, ), dtype='S10')
```

Comment:
Yeah, I'd like to avoid format changes in this PR.  And the multipole versioning might go away entirely if it's merged into the incident neutron files.

I do also plan to turn some more multipole variables like formalism and pseudo_K0RS into attributes later on.